### PR TITLE
feat(jest): 91% coverage for Iridium/utils

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖

Self-explanatory

**Which issue(s) this PR fixes** 🔨

AP-2130

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by stavares843** [comment]
this needs to be added in iridium repo instead, please add this on that repo

File: libraries/Iridium/utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { setInObject } from './utils'
+
+describe('', () => {
```
Comment (stavares843) [inline libraries/Iridium/utils.test.ts:3]: can we add a name on the describe instead of being empty?

**Review by stavares843 — CHANGES_REQUESTED** [review]

File: libraries/Iridium/utils.test.ts (hunk 1 above)
Comment (aewing) [reply libraries/Iridium/utils.test.ts:3]: I second this, as a dev it's really helpful if the test text is descriptive enough to tell me what this function was testing specifically so I can go straight to the code that it impacts without needing to read the test file.

Comment (aewing) [reply libraries/Iridium/utils.test.ts:3]: (thought I should read the test file anyway)

**Comment by drepram** [comment]
After I committed my changes, 3 snapshots of 1 test suite (`libraries/WebRTC/encoders.test.ts`) failed. If this error is reproducible in other people's machine I see that we must open a ticket to resolve the error.

I do not see the relation between the file in this PR with that test that's returning an error, but I find the use in bringing it up so that we may resolve it quickly.

**Comment by stavares843** [comment]
Captura de ecrã 2022-08-18, às 12 06 07

is all green on CI and locally not sure how you got that error

**Review by stavares843 — APPROVED** [review]

**Pull request merged**